src/main/java/duke/tasklist/Tasklist.java

```diff
@@ -0,0 +1,33 @@
+package duke.tasklist;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+public class Tasklist {
+    public ArrayList<Task> tasks;
+
+    public Tasklist() {
+        tasks = new ArrayList<>(100); // Specification said numTasks < 100.
```

Comment:
It's good to put the specification into consideration in your codes, and I didn't even notice it on my end. 💯